docs/content/preview/deploy/multi-dc/async-replication/async-replication-transactional.md

```diff
@@ -0,0 +1,62 @@
+headContent: Transactional (active-standby) replication
+menu:
+  preview:
+    parent: async-replication
+    identifier: async-replication-transactional
+    weight: 20
+type: docs
+---
+
+A transactional xCluster deployment preserves and guarantees transactional atomicity and global ordering when propagating change data from one universe to another, as follows:
+
+- Transactional atomicity guarantee. A transaction spanning tablets A and B will either be fully readable or not readable at all on the target universe. A and B can be tablets in the same table OR a table and an index OR different tables.
+
+- Global ordering guarantee. Transactions are visible on the target side in the order they were committed on source.
+
+Due to the asynchronous nature of xCluster replication, this deployment comes with non-zero recovery point objective (RPO) in the case of a source universe outage. The actual value depends on the replication lag, which in turn depends on the network characteristics between the regions.
+
+The recovery time objective (RTO) is very low, as it only depends on the applications switching their connections from one universe to another. Applications should be designed in such a way that the switch happens as quickly as possible.
+
+Transactional xCluster support further allows for the role of each universe to switch during planned and unplanned failover scenarios.
+
+The xCluster role is a property with values ACTIVE or STANDBY that determines and identifies the active (source) and standby (target) universes:
+
+- ACTIVE: The active universe serves both reads & writes. Reads/writes happen as of the latest time and according to the chosen isolation levels.
+- STANDBY: The standby universe is meant for reads only. Reads happen as of xCluster safe time for the given database.
+
+xCluster safe time is the transactionally consistent time across all tables in a given database at which Reads are served. In the following illustration, T1 is a transactionally consistent time across all tables.
+
+![Transactional xCluster](/images/deploy/xcluster/xcluster-transactional.png)
+
+## Limitations
+
+- Tablet splitting is not supported with Transactional Atomicity and Global Ordering.
+- Supports only Active-Standby setups with transactional atomicity and global ordering.
+- Transactional consistency is currently not supported for YCQL, only for YSQL.
+
+## Recommended guardrails
+
+- CPU utilisation: keep below 65%.
+- Disk space utilisation: under 65%.
+
+## Prerequisites
+
+- Create Primary and Standby Universes with TLS enabled.
+
+- Set the following flags on both Primary and Standby:
+
+  - YB-TServer
+
+    - [log_min_seconds_to_retain](../../../../reference/configuration/yb-tserver/#log-min-seconds-to-retain) = 86400
+
+        This value depends on how long a network partition or standby cluster outage can be tolerated and the amount of WAL expected to be generated during that period.
+
+  - YB-Master
```

Comment:
This whole section is no longer needed and can be removed. We no longer need the flags on the Master.

Reply:
enable_automatic_tablet_splitting and enable_automatic_tablet_splitting dont need to be set anymore.